numpy/random/mtrand/mtrand.pyx

```diff
@@ -1457,7 +1457,7 @@ cdef class RandomState:
 
         See Also
         --------
-        random.randint : Similar to `random_integers`, only for the half-open
+        np.random.randint : Similar to `random_integers`, only for the half-open
```

Comment:
Looking the working links in `choice` and `uniform`, a simple reference to `randint` without the `random.` part works.

Reply:
@charris Thanks for your review. I have incorporated your suggestion and made the change.